src/mantle/mantle_extension_discovery.c

```diff
@@ -13,6 +13,8 @@ GR_RESULT GR_STDCALL grGetExtensionSupport(
     } else if (strcmp(pExtName, "GR_BORDER_COLOR_PALETTE") == 0) {
         // TODO implement
         return GR_SUCCESS;
+    } else if (strcmp(pExtName, "GR_DMA_QUEUE") == 0) {
```

Comment:
does the game boot when we advertise the extension but don't expose the queue? unfortunately, some drivers don't expose a transfer-only queue (such as RADV). we need to take this into account